### DISCUSSION TITLE
My contributions are not showing on github graph

### DISCUSSION THREAD

**Discussion by shahalam68** [community/community#88797] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My contribution is not showing in the GitHub contribution graph. It only shows when I create a new repository. But when I push again in that same repository the code is updated but the graph is not updating. I faced the same issue a few days ago and then I updated the primary email after that the issue was solved but the same issue appeared again. But my email settings are the same as the GitHub articles say.

How do you think I could solve this problem?

**Comment by fury-05** [comment]
If you code is updates and has commits then I will suggest you try copying your profile URL and pasting in Incognito Tab and check if the graph is updated their, usually due to cache policy of browser it takes time depending upon how much TTL is set for browser cache.

Let me know about results.

**Reply by shahalam68** [reply]
It's solved after deleting the catch. And Browsing data thank you.

**Comment by ShamcyBens** [comment]
I think giving it time and also refreshing your browser might just fix that, I tried it and it worked on my side

**Comment by shahalam68** [comment]
It's solved after deleting the catch. And Browsing data.
